src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,28 @@
+import { PropsWithChildren } from 'react';
+import { Container, ContainerProps } from '@mantine/core';
+import { useElementSize } from '@mantine/hooks';
+
+import { FixedContent } from './FixedContent';
+import { GrowingContent } from './GrowingContent';
+import { LayoutContext } from './LayoutContext';
+
+export function Layout({ children, ...props }: ContainerProps & PropsWithChildren) {
+  const { ref: parentRef, height: parentHeight } = useElementSize();
+  const { ref: fixedContentRef, height: fixedContentHeight } = useElementSize();
+
+  return (
+    <LayoutContext.Provider
+      value={{
+        growingSizeHeight: parentHeight - fixedContentHeight,
+        ref: fixedContentRef
+      }}
+    >
+      <Container ref={parentRef} h={'100%'} {...props} px={'xs'} py={'md'}>
```

Comment:
I suggest we remove any build in padding from this component, and replace this with e.g: 
```suggestion
      <Boxref={parentRef} h={'100%'} {...props}>
```
Note that the container props should then be replaced with BoxProps

While it could be considered nice that there is already some styling applied to the Layout component, I think that we will end up in situations where we think it's limiting/annoying and will want to work around this. There is already a place where it's like that, in the scene menu.  Better to not include any padding at all in this component and leave i up to the user if a container should be added around it :)